packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -28,7 +28,8 @@ import type { Properties as OverlayProperties } from '@js/ui/overlay';
 import type dxOverlay from '@js/ui/overlay';
 import Overlay from '@js/ui/overlay/ui.overlay';
 import Scrollable from '@js/ui/scroll_view/ui.scrollable';
-import { current as currentTheme, isMaterialBased } from '@js/ui/themes';
+// @ts-expect-error
```

Comment:
That's not okay that we add new "@ts-expect-error".
I have no idea why `isGeneric` is still not added to .d.ts. I agree to do it in a separate task to not involve API review to this bug fix. But please create a card for this task.